src/azul/plugins/metadata/hca/indexer/transform.py

```diff
@@ -1693,7 +1694,7 @@ def _transform(self) -> Transform:
                         contributed_analyses=contributed_analyses,
                         dates=[self._date(self._singleton_entity())],
                         projects=[self._project(self._api_project)])
-        return self._add_replica(contents, self._singleton_entity())
+        return self._add_replica(contents, self._singleton_entity(), list(map(str, visitor.files)))
```

Comment:
L 

Please extract variable as requested in previous review, for all calls to _add_replica